akka-stream/src/main/resources/reference.conf

```diff
@@ -45,6 +45,16 @@ akka {
       # this may cause an initial runtime overhead, but most of the time fusing is
       # desirable since it reduces the number of Actors that are created.
       auto-fusing = on
+      
+      # Those stream elements which have explicit buffers (like mapAsync, mapAsyncUnordered,
+      # buffer, flatMapMerge, Source.actorRef, Source.queue, etc.) will preallocate a fixed
+      # buffer upon stream materialization if the requested buffer size is less than this
+      # configuration parameter. The default is very high because failing early is better
+      # than failing under load.
+      #
+      # Buffers larger than this value will use a dynamic approach that has a higher
+      # per-element cost.
+      max-fixed-buffer-size = 1000000000
```

Comment:
Fair enough. You might want to loosen the wording on "Buffers larger than this value will use a dynamic approach that has a higher per-element cost." to something like: "Buffers sized larger than this will dynamically grow/shrink and may consume more memory—per element—than the fixed size buffers."